Tools/Release/update_dependencies.py

```diff
@@ -108,7 +110,7 @@ def update(args):
         if repo_name != "warpx":
             print(f"- old commit: {dependencies_data[commit_key]}")
             print(f"- new commit: {new_commit_sha}")
-            if dependencies_data[commit_key] == repo_commit_sha:
+            if dependencies_data[commit_key] == new_commit_sha:
```

Comment:
`new_commit_sha` is the variable that contains the correct information (commit sha or version tag).